_posts/2024-10-15-sunsetting-uswds-1-0.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Sunsetting the USWDS 1.0 documentation site
+tags:
+- product update
+category: About
+excerpt: It's been over five years since we launched USWDS 2.0 and moved on from USWDS v1. We archived the USWDS v1 documentation site a few years back, and at the end of 2024 we'll be taking it offline. Content will likely still be available via archive.org.
+excerpt_link_url: https://v1.designsystem.digital.gov/
```

Comment:
**quibble**: Feels confusing to say we're sunsetting it and then send them to the page.

Reply:
Hi James, we'd linked to it in previous newsletters and monthly calls - to give users an opportunity to review and let us know if there were things they wanted to keep. Is there anywhere you can recommend we guide them instead? @amyleadem I'm also wondering if there might be instances where we don't link to anything from an announcement...?

Reply:
It is an interesting thing to consider and definitely worth discussing.

I can code in an option that would let us turn off the link in the header. Here is a preview of what this post would look like without a header link:

<img width="811" alt="image" src="https://github.com/user-attachments/assets/2e0a2ed9-4810-4da2-b3e2-03bcca944ae6">

Let me know if you want me to include this as an option.

Reply:
I think I do, for discussion.  _And_ I'm wondering about this post - as it ages and we take down that site, we'll need to remove that link, right?

Reply:
A potentially new use case worth discussing. FWIW, I don't see this question as a blocker for this PR.